### ISSUE TITLE
CGP103: New course data

### ISSUE THREAD

**Issue by lebaz20** [camelcasetechsd/certigate#103]
Add New Resources for courses
Type : Standards[one file], Course Updates[multi files], Ice Breakers[one file]

**Comment by lebaz20** [comment]
- [x] Add new types in form
- [x] JS Validation : hide add more button in case of standards or ice breakers types for resources
- [x] Server-side Validation : check if more than one standards or ice breakers for resources are submitted or any of them already exists

**Comment by lebaz20** [comment]
Branch is CGP102_New_Course_Data ,Commits used by mistake #102 as bug id instead of #103

**Comment by babanesma** [comment]
- merge development branch into this branch to retrieve the new theme

**Comment by babanesma** [comment]
- Fatal error appears when creating or editing course events 

```
( ! ) Fatal error: Undefined class constant 'TYPE_BOTH' in /var/www/html/module/Courses/src/Courses/Form/CourseEventForm.php on line 94
```
- when logging in with admin , the admin menu is enlarging the header and covering the new buttons of creating new events ( I think this is happening with everything else ) 
  Resolve this when logging with admin only

**Comment by lebaz20** [comment]
@babanesma First issue 'Fatal error' is fixed in #105 as it is not related to current task scope
